### DISCUSSION TITLE
Fully sampled version of knee_singlecoil_test dataset

### DISCUSSION THREAD

**Discussion by marinapapageorgiou** [facebookresearch/fastMRI#351] [Q&A]
Hello, 
How can I find the fully sampled version of knee_singlecoil_test dataset, so that I can obtain the ground truth and calculate the PSNR and SSIM?

**Comment by mmuckley** [comment]
Hello @marinapapageorgiou, I don't think this was released. We don't manage the data here. That is done by NYU. So you'll have to email [fastmri@med.nyu.edu](https://fastmri.med.nyu.edu/fastmri@med.nyu.edu).
